src/main/java/edu/greenblitz/robotName/subsystems/swerve/Modules/mk4iSwerveModule/MK4iSwerveConstants.java

```diff
@@ -26,6 +26,7 @@ public class MK4iSwerveConstants {
 	public static final double ANGLE_TICKS_TO_WHEEL_TO_RPM = RobotConstants.General.Motors.FALCON_VELOCITY_UNITS_PER_RPM / ANGULAR_GEAR_RATIO;
 	public static final double LINEAR_TICKS_TO_METERS_PER_SECOND = RobotConstants.General.Motors.FALCON_VELOCITY_UNITS_PER_RPM / LINEAR_GEAR_RATIO * WHEEL_CIRCUMFERENCE / 60;
 	public static final double ANGLE_TICKS_TO_RADIANS = RobotConstants.General.Motors.FALCON_TICKS_PER_RADIAN / ANGULAR_GEAR_RATIO;
+	public static final double COUPLING_RATIO = 1.0;
```

Comment:
in a file called MK4ISwerveConstants you would expect the constants to be of the mk4i module

Reply:
You're god damn right

Reply:
i'll find this ration tommorow :)